common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -41,19 +41,43 @@ public ConceptService(EntityManager entityManager) {
   public Slice<Concept> searchConcepts(String query,
       StandardConceptFilter standardConceptFilter, List<String> vocabularyIds,
       List<String> domainIds, int limit) {
+
+      //String[] keywords=query.split("\\s*(=>|,|\\s)\\s*");
+      String[] keywords=query.split("[,+\\s+]");
```

Comment:
Can you add a comment indicating what we are doing here? Perhaps put this logic into a separate helper function?

Reply:
I like comments. If we will use this on other searches which we might -- we need to do the ppi search this way too, make a function.